oclmath/reference_math.cpp

```diff
@@ -2469,6 +2472,7 @@ static inline void mul128( cl_ulong a, cl_ulong b, cl_ulong *hi, cl_ulong *lo )
 
 // Move the most significant non-zero bit to the MSB
 // Note: not general. Only works if the most significant non-zero bit is at MSB-1
+/*
```

Comment:
Please, do not leave commented out code. Please, remove it if it's not used by CTS.

Reply:
This code wasn't used by anything, that's why I got a linker warning. (Unreferenced function). However the logic seemed tricky/hard enough to cause trouble when trying to reproduce. It's still in Git history, but it's less apparent to a future user that at one point we had such a function. My rationale in commenting was trying to avoid someone reinventing this wheel in the future, but it can be removed if you insist.

Reply:
It's better to remove it IMO. We can add it somewhere for reference (a GitHub issue?).